security/index.adoc

```diff
@@ -33,17 +33,13 @@ endif::[]
 [[security-intro-what-are-containers]]
 == What Are Containers?
```

Comment:
I'm still not sure this section is needed. Would it be better housed in the Architecture Guide?